Library/Formula/racer.rb

```diff
@@ -0,0 +1,41 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+
+  keg_only "You should set path in your special editor which use racer"
```

Comment:
Wouldn't ```libexec``` be more appropriate than ```opt_prefix```? This is a binary that is not called directly but by other programs (editors): http://www.linuxbase.org/betaspecs/fhs/fhs/ch04s07.html